README.md

```diff
@@ -7,10 +7,10 @@
 ## About
 
 Typeshed contains external type annotations for the Python standard library
-and Python builtins, as well as third party packages as contributed by
+and Python builtins, as well as third-party packages that are contributed by
```

Comment:
This sounds ungrammatical to me, should be "that have been", or just omit the "that are" part.